writeToStream.js

```diff
@@ -0,0 +1,579 @@
+
+'use strict';
+
+var protocol = require('./constants')
+  , empty = new Buffer(0)
+  , zeroBuf = new Buffer([0])
+  , numCache = require('./numbers')
+
+function generate(packet, stream) {
+
```

Comment:
Commenting w/ partial context, but using `nextTick()` is how we `uncork()` in the `'http'` module.